printer_mods/ppak10/Cable_Port_Flush_Mount_Skirt/README.md

```diff
@@ -0,0 +1,34 @@
+# Cable Port Flush Mount Skirt
+
+Provides an inlet to mount BATIGE circular cable adapter ports on the rear
+center skirt (250, 300, and 350) allowing easy access to display and USB headers
+of the raspberry pi.
+
+It can be used for quick access to Raspberry Pi ports often out of reach
+underneath the printer without needing to flip over the printer.
+
+<div style="justify-content: center; display: flex;">
+  <img src="./Images/250_300_350_Render.png" alt="CAD" width="800"/>
+</div>
+
+<div style="justify-content: center; display: flex;">
+  <img src="./Images/250.jpeg" alt="250" width="250"/>
+  <img src="./Images/300.jpeg" alt="300" width="250"/>
+  <img src="./Images/350.jpeg" alt="350" width="250"/>
+</div>
+
+<div style="justify-content: center; display: flex;">
+  <img src="./Images/350_Installed.jpeg" alt="350 Installed" width="800"/>
+</div>
+
+## Usage
+This example uses the [Micro-HDMI and
+USB-C](https://www.amazon.com/dp/B0C68MPKHV?ref=ppx_yo2ov_dt_b_product_details&th=1)
+variant of the BATIGE Flush Mount, however other this mod is suitable for other
+variants that satisfy a [similar form factor](https://www.amazon.com/stores/BATIGE/page/ADB1F3D7-7733-487F-B5AB-6F021FF57798?ref_=ast_bln).
+
```

Comment:
up to you, but you _may_ want to add that it is a friction fit with no additional hardware required besides the BATIGE port.